Zero-Gravity-Store-React/src/Pages/AuthWidget/SignUp/SignUp.js

```diff
@@ -0,0 +1,66 @@
+// import "./Login.css";
+import { Link } from "react-router-dom";
+import logo from "../../../Asserts/Images/hero-logo.png";
+export function SignUp (){
+    return(
+        <div className="html">
+        <div className="background">
+
+        <div className="login-box">
+        <div className="login-content">
+            <div className="auth-logo">
+                <div className="logo-mg">
+                  <img className="logo" src={logo} alt="logo" />
+                </div>
+                <div className="bg-color">
+                  ZERO
+                  <div>GRAVITY</div>
+                  <small className="small-txt">Store</small>
+                </div>
+              </div>
+              <span className="pd">Sign Up with 0`Gravity Account</span>
+              <div className="login-credals">
+                <form action="#" method="#" className="form login">
+
+                    <div className="form__field">
+                      <label className="auth-label" for="login__username"><span className="auth-icon"><i className="fa fa-envelope"></i></span><span className="hidden">First name</span></label>
+                      <input id="login__username" type="text" name="username" className="form__input log-in-input" placeholder="First name" required />
+                    </div>
```

Comment:
Rather than repeating this, you can make a general component and use that by passing props according to field